backend/api/users.py

```diff
@@ -172,4 +174,14 @@ def get_bookmarks():
     if not res:
         return make_response("Unknown User with Email-address: " +
                              user['email'], 400)
+    try:
+        for project in res:
+            project['is_bookmark'] = 'true' if str(project['_id'])\
+                in current_user['bookmarks'] else 'false'
+            project['is_owner'] = 'true' if current_user['email']\
+                in [author['email'] for author in project['authors']]\
+                else 'false'
+
+    except KeyError:
+        pass
```

Comment:
but "res" is a user object, there are no "projects" and it's not list-iterable but dict-iterable, returning name, email, password, etc.

The call only returns a bookmark id